security/netbird/src/opnsense/mvc/app/controllers/OPNsense/NetBird/StatusController.php

```diff
@@ -28,16 +29,16 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-namespace OPNsense\Netbird;
+namespace OPNsense\NetBird;
 
 /**
- * Class ConstatusController
- * @package OPNsense\Netbird
+ * Class StatusController
+ * @package OPNsense\NetBird
  */
-class ConstatusController extends \OPNsense\Base\IndexController
+class StatusController extends \OPNsense\Base\IndexController
 {
     public function indexAction()
     {
-        $this->view->pick('OPNsense/Netbird/constatus');
+        $this->view->pick('OPNsense/NetBird/status');
```

Comment:
Look, I changed the namespace for clarity reasons. Can we not do branding on the file system? You can change all user-visible strings to NetBird, we do the same for WireGuard. In some cases doing Pascal case you end up clashing with API calls that will look like /api/net_bird/xxx instead of /api/netbird/xxx